### DISCUSSION TITLE
DATEV Rechnungsdatenservice 1.0 for self-hosted installations

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#3475] [Q&A] [answered]
Hey! I read the blog post about the DATEV Rechnungsdatenservice 1.0 some while ago and read it again today. I cannot find "DATEV Datenservices" below "Export" in our self-hosted docker-based installation and didn´t find anything about it here on github. So i guess it is only available for direct pretix-customers, right? Would totally understand that! Just want to clearify. Thanks a lot!

**Comment by raphaelm** [comment, marked as answer]
Correct, it's currently only available on pretix Hosted, unfortunately, as it requires a lengthy and expensive certification process with DATEV to get an API key

**Comment by nilskamm** [comment]
Thank you for the, as always, really fast answer!
